src/pages/references/language-clarity.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Clarity Language
```

Comment:
I think this could be renamed to `Overview`. and the file to `language-overview`